src/erc20.ts

```diff
@@ -46,6 +47,18 @@ export function handleTransfer(event: TransferEvent, token: ERC20Token): void {
         tokenPosition.save()
     }
 
+    // Update airdropped amounts for the referral program
+    if (from == REFERRAL_ADDRESS) {
+        const referralProgram = loadReferralProgram(event, Address.fromString(REFERRAL_ADDRESS))
+        referralProgram.totalAirdropped = referralProgram.totalAirdropped.plus(amount)
+        referralProgram.save()
+
+        const referralPosition = loadReferralPosition(Address.fromString(REFERRAL_ADDRESS), event.params.to)
+        referralPosition.totalAirdroppedAmount = referralPosition.totalAirdroppedAmount.plus(amount)
+        referralPosition.totalRewardsPending = referralPosition.totalRewardsPending.minus(amount)
+        referralPosition.save()
+    }
```

Comment:
Does this apply only for OVL transfers? If so, I would move it to the `handleTransferOVL` handler instead.